vmaas_sync/metrics.go

```diff
@@ -120,6 +123,12 @@ func update() {
 	updateAdvisoryMetrics()
 	updateSystemAdvisoriesStats()
 	updateProcessesMetrics()
+	enableCyndiMetrics, err := strconv.ParseBool(os.Getenv("ENABLE_CYNDI_METRICS"))
```

Comment:
You can use `utils.GetBoolEnvOrDefault` to skip errors & should probably get this value at the start of the program by making it a module level variable